### ISSUE TITLE
Project Add delivers Error: "could not create project: 500"

### ISSUE THREAD

**Issue by alexrjs** [ansible-semaphore/semaphore#83]
Error:

> Error 1054: Unknown column 'created' in 'field list

Log:
```
...
[GIN] 2016/04/30 - 14:07:25 | 200 |     181.097µs | 127.0.0.1 |   GET     /public/html/projects/add.html
[mysql] 2016/04/30 14:07:32 packets.go:33: unexpected EOF
[mysql] 2016/04/30 14:07:32 packets.go:33: unexpected EOF
2016/04/30 14:07:32 notifying bugsnag: Error 1054: Unknown column 'created' in 'field list'
2016/04/30 14:07:32 bugsnag.Notify: not notifying in development

2016/04/30 14:07:32 [Recovery] panic recovered:
POST /api/projects HTTP/1.1
Host: localhost:3000
Accept: application/json, text/plain, */*
Accept-Encoding: gzip, deflate
Accept-Language: de-DE,de;q=0.8,en-US;q=0.6,en;q=0.4,it;q=0.2
Cache-Control: no-cache
Connection: keep-alive
Content-Type: application/json;charset=UTF-8
Cookie: semaphore=uW95nBPS_WYanIBQ3Yrz-La5mM0VvoR81narqVSYCII%3D
Origin: http://localhost:3000
Referer: http://localhost:3000/
User-Agent: Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/50.0.2661.75 Safari/537.36

Error 1054: Unknown column 'created' in 'field list'
/usr/local/Cellar/go/1.6/libexec/src/runtime/panic.go:426 (0x42da79)
/Users/m/p/gospace/src/github.com/bugsnag/bugsnag-go/bugsnag.go:52 (0x513261)
/usr/local/Cellar/go/1.6/libexec/src/runtime/asm_amd64.s:472 (0x45cd6e)
/usr/local/Cellar/go/1.6/libexec/src/runtime/panic.go:426 (0x42da79)
/Users/m/p/gospace/src/github.com/ansible-semaphore/semaphore/routes/projects/projects.go:42 (0x6d23ac)
/Users/m/p/gospace/src/github.com/gin-gonic/gin/context.go:97 (0x51c31a)
/Users/m/p/gospace/src/github.com/ansible-semaphore/semaphore/routes/auth.go:105 (0x4b105b)
/Users/m/p/gospace/src/github.com/gin-gonic/gin/context.go:97 (0x51c31a)
/Users/m/p/gospace/src/github.com/ansible-semaphore/semaphore/routes/auth.go:96 (0x4b0a7f)
/Users/m/p/gospace/src/github.com/gin-gonic/gin/context.go:97 (0x51c31a)
/Users/m/p/gospace/src/github.com/gin-gonic/gin/logger.go:66 (0x52dc4a)
/Users/m/p/gospace/src/github.com/gin-gonic/gin/context.go:97 (0x51c31a)
/Users/m/p/semaphore/main.go:74 (0x401b5f)
/Users/m/p/gospace/src/github.com/gin-gonic/gin/context.go:97 (0x51c31a)
/Users/m/p/gospace/src/github.com/gin-gonic/gin/recovery.go:45 (0x52e9b1)
/Users/m/p/gospace/src/github.com/gin-gonic/gin/context.go:97 (0x51c31a)
/Users/m/p/gospace/src/github.com/gin-gonic/gin/gin.go:284 (0x522f72)
/Users/m/p/gospace/src/github.com/gin-gonic/gin/gin.go:265 (0x522ba7)
/usr/local/Cellar/go/1.6/libexec/src/net/http/server.go:2081 (0x67ca9e)
/usr/local/Cellar/go/1.6/libexec/src/net/http/server.go:1472 (0x67934e)
/usr/local/Cellar/go/1.6/libexec/src/runtime/asm_amd64.s:1998 (0x45f421)
...
```
![image](https://cloud.githubusercontent.com/assets/10467960/14935792/cc2168a8-0edd-11e6-83d8-ec428efbd2b5.png)

**Comment by matejkramny** [comment]
@alexrjs will be released with next beta

**Comment by dsmorse** [comment]
I'm still seeing this issue on the 2.0 final in both the linux_386 and _amd64


2016/05/19 20:16:25 [Recovery] panic recovered:                                                                                                                                                                                              
POST /api/projects HTTP/1.1                                                                                                                                                                                                                  
Host: localhost:8080                                                                                                                                                                                                                         
Accept: application/json, text/plain, */*                                                                                                                                                                                                    
Accept-Encoding: gzip, deflate                                                                                                                                                                                                               
Accept-Language: en-US,en;q=0.8                                                                                                                                                                                                              
Cache-Control: no-cache                                                                                                                                                                                                                      
Connection: keep-alive                                                                                                                                                                                                                       
Content-Type: application/json;charset=UTF-8                                                                                                                                                                                                 
Cookie: AJS.conglomerate.cookie="|SPC-SELECTED-CATEGORY-BFGD=1|SPC-SELECTED-CATEGORY-AS=1|SPC-SELECTED-CATEGORY=all"; csrftoken=LLinXRKXRfElAGJ01gp2AF91Z1KzdOkX; __utma=111872281.1383182836.1462419672.1462419672.1462419672.1; __utmz=1118
72281.1462419672.1.1.utmcsr=(direct)|utmccn=(direct)|utmcmd=(none); iconSize=16x16; hudson_auto_refresh=false; JSESSIONID.fb158369=17tmqoltxhi231olv5e71p37fb; JSESSIONID.5daf8c18=1xk5ebqc10f6fc9ar1zc90ujl; screenResolution=1920x1080; _se
ssion_id=BAh7CkkiCXVzZXIGOgZFRmkISSIPZXhwaXJlc19hdAY7AEZsKwerFz9XSSIPc2Vzc2lvbl9pZAY7AEZJIiU4ODY4YTcwOWYxNGQ1MWE0MWUxNzI4NzhlN2YwZWU0NwY7AFRJIgtsb2NhbGUGOwBGIgdlbkkiEF9jc3JmX3Rva2VuBjsARkkiMS9RbVBMUDY5NzdDOEM2UUQ2ZUxvWXVIUDNuMXZrUWhsaVdp
bnY1emxUZFU9BjsARg%3D%3D--34f525480afca4523a6fa10e228a5e1dbe87f232; JSESSIONID=cpp16yjgnq0xruy9pj19cbi5; semaphore=MTQ2MzcxMDIyM3xJWWNyOHJBWXZpajRxbV9KNkNQQ0l0UXlyQ250UUhmcWxka1NybTdhUk51NmgxRmFvc0dRYXpmWUJvOGRQYjRTcnpSXzVndWQzXzdVenh4aE
F5OTBoZW89fGvRjmIGBGrc9u643i1L2bl_HUR45d23FCGKIcctx6mV                                                                                                                                                                                       
Origin: http://localhost:8080                                                                                                                                                                                                                
Referer: http://localhost:8080/                                                                                                                                                                                                              
User-Agent: Mozilla/5.0 (Windows NT 6.1; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/50.0.2661.94 Safari/537.36                                                                                                                     
                                                                                                                                                                                                                                             
                                                                                                                                                                                                                                             
Error 1364: Field 'created' doesn't have a default value                                                         
/usr/local/Cellar/go/1.6/libexec/src/runtime/panic.go:426 (0x8071e6d)                                            
/Users/m/p/gospace/src/github.com/bugsnag/bugsnag-go/bugsnag.go:52 (0x814142c)                                   
/usr/local/Cellar/go/1.6/libexec/src/runtime/asm_386.s:488 (0x809934a)                                           
/usr/local/Cellar/go/1.6/libexec/src/runtime/panic.go:426 (0x8071e6d)                                            
/Users/m/p/gospace/src/github.com/ansible-semaphore/semaphore/routes/projects/projects.go:38 (0x829487c)         
/Users/m/p/gospace/src/github.com/gin-gonic/gin/context.go:97 (0x8148939)                                        
/Users/m/p/gospace/src/github.com/gin-gonic/gin/logger.go:66 (0x81568d6)                                         
/Users/m/p/gospace/src/github.com/gin-gonic/gin/context.go:97 (0x8148939)                                        
/Users/m/p/semaphore/main.go:73 (0x8049929)                                                                      
/Users/m/p/gospace/src/github.com/gin-gonic/gin/context.go:97 (0x8148939)                                        
/Users/m/p/gospace/src/github.com/gin-gonic/gin/recovery.go:45 (0x81573ca)                                       
/Users/m/p/gospace/src/github.com/gin-gonic/gin/context.go:97 (0x8148939)                                        
/Users/m/p/gospace/src/github.com/gin-gonic/gin/gin.go:284 (0x814e063)                                           
/Users/m/p/gospace/src/github.com/gin-gonic/gin/gin.go:265 (0x814dd5e)                                           
/usr/local/Cellar/go/1.6/libexec/src/net/http/server.go:2081 (0x82497c6)                                         
/usr/local/Cellar/go/1.6/libexec/src/net/http/server.go:1472 (0x8246a58)                                         
/usr/local/Cellar/go/1.6/libexec/src/runtime/asm_386.s:1585 (0x809b161)

**Comment by matejkramny** [comment]
Related to #92

**Comment by alexrjs** [comment]
Now v2.0 works on my linux: Linux linix 4.2.0-36-generic #42-Ubuntu SMP Thu May 12 22:05:35 UTC 2016 x86_64 x86_64 x86_64 GNU/Linux

So my original bug is fixed. 

![bildschirmfoto vom 2016-05-20 20-04-54](https://cloud.githubusercontent.com/assets/10467960/15437332/c9bd84d6-1ec6-11e6-88f4-fe5c85997a98.png)

**Comment by matejkramny** [comment]
Thats good, but other users are reporting the same bug. I'll investigate and fix if needed.

What is your feedback overall?

**Comment by matejkramny** [comment]
Fixed with #977835f92354a37786b582114ff525ea4e5c22b4

**Comment by alexrjs** [comment]
Thanks for you great work and the fixes.

Overall experience was good. Look/Layout is clean an easy to understand. Maybe the wiki needs a little more polish and integration into the interface (I know it is hidden in the system info menu, but a '?' - menu beside the '*' in the top menu would be more intuitive)... But as a developer myself I know documentation is not always the highest priority :-)
